pkg/boot/esxi/esxi_test.go

```diff
@@ -311,3 +313,25 @@ func TestImageOrder(t *testing.T) {
 		t.Fatalf("getImages(%s, %v, %v) = %v, want %v", device, opt5, opt6, imgs, want)
 	}
 }
+
+func FuzzParse(f *testing.F) {
+	seeds, err := filepath.Glob("testdata/*.cfg")
+	if err != nil {
+		f.Errorf("failed to find seed corpora files: %v", err)
```

Comment:
(here and other files below) I think `t.Fatalf` here please, because if `err` is not nil then the value of `seeds` is indeterminate so it doesn't make sense to keep running the rest of the code.

Reply:
Yeah, in the context of fuzzing it is okay to fail getting the seed corpora. But this might seem flaky in the CI context if tests are just getting skipped without any actual inputs being run. 
Will push a fix tomorrow.

Reply:
I updated all the fuzzing tests with e324725